medical-practitioner-portal/src/app/case-submissions/case-submissions.component.ts

```diff
@@ -29,8 +30,20 @@ import {
     MatExpansionPanelHeader,
     MatExpansionPanelTitle,
   ],
+  providers: [DatePipe],
   templateUrl: './case-submissions.component.html',
   styleUrl: './case-submissions.component.scss',
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
 })
-export class CaseSubmissionsComponent {}
+export class CaseSubmissionsComponent {
+  @Input() documents: CaseDocument[] = [];
+
+  constructor(private datePipe: DatePipe) {}
+
+  getFormattedDate(date: string | undefined | null) {
```

Comment:
this is fine for now but when we need this method elsewhere, move it to a service that we can reuse

Reply:
I added this as TODO item